aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -49,6 +49,7 @@ func resourceAwsApiGatewayRestApi() *schema.Resource {
 			"policy": {
 				Type:             schema.TypeString,
 				Optional:         true,
+				Computed:         true,
```

Comment:
This update breaks the existing `TestAccAWSAPIGatewayRestApi_policy` test:

```
=== CONT  TestAccAWSAPIGatewayRestApi_policy
TestAccAWSAPIGatewayRestApi_policy: resource_aws_api_gateway_rest_api_test.go:446: Step 4/4 error: Check failed: Check 1/1 error: aws_api_gateway_rest_api.test: Attribute 'policy' expected "", got "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Deny\",\"Principal\":{\"AWS\":\"*\"},\"Action\":\"execute-api:Invoke\",\"Resource\":\"*\"}]}"
--- FAIL: TestAccAWSAPIGatewayRestApi_policy (199.92s)
```

Since the removal behavior is expected unless it is configured to an empty string (`""`), will update the test and document the behavior in the resource documentation on merge.